gps/index.html

```diff
@@ -10,27 +10,27 @@
 
 
-<h2 class="text-left"><i class="fas fa-fw fa-user-plus"></i> Who should be using a Network</h2>
-<p>Any person or company that wants or requires precision GPS information should be using a GPS Network. The most common applications including surveying, engineering, construction, GPS controlled machinery and GIS data collection.</p>
+<h2 class="text-left"><i class="fas fa-fw fa-user-plus"></i> Who Should Be Using a Network?</h2>
+<p>Any person or company that wants or requires precision GPS information should be using a GPS network. The most common applications of a GPS network include surveying, engineering, construction, GPS-controlled machinery, and GIS data collection.</p>
 
-<h2 class="text-left"><i class="fas fa-fw fa-signal"></i> Reference Network Benefits</h2>
+<h2 class="text-left"><i class="fas fa-fw fa-signal"></i> What Are the Benefits of Using a Reference Network?</h2>
 <img src="images/accuracy.png" class="outline flex flex--center" style="margin-bottom:2em;width:100%;max-width:778px">
-<p>A reference network eliminates the need for a reference (base) station, therefore reducing your costs. They minimizes Ionospheric errors for rovers in the network coverage. And improve efficiency and productivity in your surveys.</p>
+<p>A reference network eliminates the need for a reference (base) station, therefore reducing your costs. Reference networks minimize ionospheric errors for rovers in the network coverage and improve efficiency and productivity in your surveys.</p>
 <ul>
-  <li>View the <a href="https://broadband.utah.gov/map/" rel="noopener" target="_blank">Cell coverage in Utah</a>
-  <li>View the <a href="http://www.connectnv.org/interactive-map" rel="noopener" target="_blank">Cell coverage in Nevada</a>
+  <li>View the <a href="https://broadband.utah.gov/map/" rel="noopener" target="_blank">cell coverage in Utah</a>
+  <li>View the <a href="http://www.connectnv.org/interactive-map" rel="noopener" target="_blank">cell coverage in Nevada</a>
 </ul>
 
-<h2 class="text-left"><i class="fas fa-fw fa-location-arrow"></i> Where are the Reference Networks Near Me</h2>
-<p>Currently, there are two networks that we are associated with.
-TURN GPS Network which serves the State of Utah and some areas of surround states with 95 sensors.
-Nevada GPS which servers the greater Reno Nevada area, with 12 sensors.
-For further information select the appropriate link below.
+<h2 class="text-left"><i class="fas fa-fw fa-location-arrow"></i> Where Are the Reference Networks Near Me?</h2>
+<p>Currently, AGRC is associated with two networks:
```

Comment:
Is this what we mean here? "AGRC is associated with"?

Reply:
yes that is correct